pkgs/by-name/fl/flaci/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  makeWrapper,
+  electron,
+}:
+
+buildNpmPackage rec {
+  pname = "flaci";
+  version = "unstable-2024-12-10";
+
+  src = fetchFromGitHub {
+    owner = "gitmh";
+    repo = "FLACI";
+    rev = "cc03bb2b4c8cf205d04936cc2110acd34397b789";
+    hash = "sha256-Wgkxlz4d1KNoPL9290D1qIbOqUYW/iAIWE9YL35Tfno=";
+  };
+
+  # Make dependency install not fail because the electron install wants to download a binary
+  ELECTRON_SKIP_BINARY_DOWNLOAD = 1;
+  npmDepsHash = "sha256-6X4R6D9GAe4DiIXCT62xKbHhLqdTE/BLlXJQKwt8oaQ=";
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+  dontNpmBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,share/flaci}
+    cp -r appmenu.html favicon.ico flaci.html flaci.js flaciLogo.png i18n images index.html languageselect.html libs main.js package.json preload.js renderer.js styles views $out/share/flaci/
```

```suggestion
    cp -r * $out/share/flaci/
```
Ideally we don't hardcode such a long list of paths